contracts/base/ModuleManager.sol

```diff
@@ -55,11 +68,15 @@ contract ModuleManager is SelfAuthorized, Executor {
         emit DisabledModule(module);
     }
 
-    /// @dev Allows a Module to execute a Safe transaction without any further confirmations.
-    /// @param to Destination address of module transaction.
-    /// @param value Ether value of module transaction.
-    /// @param data Data payload of module transaction.
-    /// @param operation Operation type of module transaction.
+    /**
+     * @notice Execute `operation` (0: Call, 1: DelegateCall) to `to` with `data` and `value` (Native Token)
```

Comment:
should we have the full data in here? might be quite long ... but not sure what a good alternative would be. At least I would exchange value and data in the text to avoid that the value gets lost after a long data.

Reply:
removed data